devtools-ci-index.yaml

```diff
@@ -276,8 +276,10 @@
     <<: *job_template_defaults
 
 - job-template:
-    name: '{ci_project}-{git_repo}-f8planner-snapshot'
-    image_name: 'MISSING_IMAGE_NAME'
```

Comment:
are we sure we re ok to lose the image_name ? I dont know what impact this will have on the job itself

Reply:
I am using this [script](https://github.com/fabric8-ui/fabric8-planner/pull/2473/files#diff-495bc62c479e4b8a1776161b0b0d62f8) to build snapshot and run tests! there I am using different variables so. I am not sure if `image_name` variable name itself has some importance or meaning.